test/functional/apps/discover/_saved_queries.ts

```diff
@@ -138,8 +136,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(await filterBar.hasFilter('extension.raw', 'jpg')).to.be(true);
         expect(timePickerValues.start).to.not.eql(PageObjects.timePicker.defaultStartTime);
         expect(timePickerValues.end).to.not.eql(PageObjects.timePicker.defaultEndTime);
-        await retry.waitFor(
+        await retry.waitForWithTimeout(
```

Comment:
Can we please change this to either the `retry.try` or find some way to log the actual count within this loop?  I can't stand that the test can fail and not output what the actual count was.

Reply:
Done.